src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -27,30 +27,44 @@ public class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
-            + "Specify whether the person is a patient or specialist using the "
-            + PATIENT_TAG + " or " + SPECIALIST_TAG + " tags. \n"
-            + "Parameters: "
+    public static final String MESSAGE_USAGE_GENERAL = "Parameters: "
```

Comment:
I believe `MESSAGE_USAGE_GENERAL` can be made private here since it is no longer used outside this class, only the `MESSAGE_USAGE_PATIENT` and `MESSAGE_USAGE_SPECIALIST` are used by external classes. (Similarly for delete, edit, and find command)

Reply:
Agreed, I have changed them to private